src/hooks/useGetQuestion.ts

```diff
@@ -0,0 +1,23 @@
+import { useAppDispatch, useAppSelector } from "../state/reduxHooks";
+import axios from "axios";
+import { useEffect } from "react";
+import { API_URL } from "../settings";
+
+export function useCreateGame() {
+  const { code, questionStartCallTs } = useAppSelector(state => state.game);
```

Comment:
I think Seb has changed this to gameCode in his newest PR so be sure to pull main and update it